### DISCUSSION TITLE
How do I upgrade from Tailwind CSS v^1.5 to v^1.8.10?

### DISCUSSION THREAD

**Discussion by emicheldev** [tailwindlabs/tailwindcss#2541] [Help] [answered]
Hi 👋,
Please, how Upgrading from v^1.5 to v^1.8.10 ?

**Comment by simonswiss** [comment, marked as answer]
Hey! 👋

## Short answer

You can just run `npm install tailwindcss@latest` to use the latest version of Tailwind CSS. There are no breaking changes since version 1.0 👍

## More details

Upgrading from v1.5 to v1.8 should be straightforward since there are no major breaking changes between these releases.

Two little deprecations to consider, if you were using them: the [deprecated gap utilities](https://tailwindcss.com/docs/release-notes#deprecations-1) in v1.7 and the [deprecated conservative purge layer](https://tailwindcss.com/docs/release-notes#deprecations) in v1.8.

Outside of that, it's mostly _additions_ to the framework, so you'll have more utilities and functionality out of Tailwind, without losing anything you were using up to now.

By the way, [v1.9 was just released](https://tailwindcss.com/docs/release-notes#tailwind-css-v1-9), so you might want to upgrade straight to that 🎉

Hope it helps! 👍
